nav_msgs/Odometry current_odom
---
occupancy_grid/LastBoxWaypoint last_box_waypoint
